cloudinit/openvpn-config.tpl.yml

```diff
@@ -13,303 +13,119 @@ write_files:
+      # client config file as well. Note that v2.4 client/server will
+      # automatically negotiate AES-256-GCM in TLS mode. See also the ncp-cipher
+      # option in the manpage
       cipher AES-256-CBC
 
-      # Enable compression on the VPN link and push the
-      # option to the client (v2.4+ only, for earlier
-      # versions see below)
+      # Enable compression on the VPN link and push the option to the client
+      # (v2.4+ only, for earlier versions see below)
       compress lz4-v2
       push "compress lz4-v2"
 
-      # For compression compatible with older clients use comp-lzo
-      # If you enable it here, you must also
-      # enable it in the client config file.
-      ;comp-lzo
-
       # The maximum number of concurrently connected
       # clients we want to allow.
       ;max-clients 100
 
-      # It's a good idea to reduce the OpenVPN
-      # daemon's privileges after initialization.
-      #
-      # You can uncomment this out on
-      # non-Windows systems.
-      ;user nobody
-      ;group nobody
-
-      # The persist options will try to avoid
-      # accessing certain resources on restart
-      # that may no longer be accessible because
-      # of the privilege downgrade.
+      # The persist options will try to avoid accessing certain resources on
+      # restart that may no longer be accessible because of the privilege
+      # downgrade.
       persist-key
       persist-tun
 
-      # Output a short status file showing
-      # current connections, truncated
-      # and rewritten every minute.
+      # Output a short status file showing current connections, truncated and
+      # rewritten every minute.
       status openvpn-status.log
 
-      # By default, log messages will go to the syslog (or
-      # on Windows, if running as a service, they will go to
-      # the "\Program Files\OpenVPN\log" directory).
-      # Use log or log-append to override this default.
-      # "log" will truncate the log file on OpenVPN startup,
-      # while "log-append" will append to it.  Use one
-      # or the other (but not both).
-      ;log         openvpn.log
-      ;log-append  openvpn.log
-
-      # Set the appropriate level of log
-      # file verbosity.
+      # Set the appropriate level of log file verbosity.
       #
       # 0 is silent, except for fatal errors
       # 4 is reasonable for general usage
       # 5 and 6 can help to debug connection problems
       # 9 is extremely verbose
       verb 3
 
-      # Silence repeating messages.  At most 20
-      # sequential messages of the same message
-      # category will be output to the log.
+      # Silence repeating messages.  At most 20 sequential messages of the same
+      # message category will be output to the log.
       ;mute 20
 
-      # Notify the client that when the server restarts so it
-      # can automatically reconnect.
+      # Notify the client that when the server restarts so it can automatically
+      # reconnect.
```

Comment:
This is awkwardly phrased.

Reply:
Feel free to submit a PR to upstream: 

https://github.com/OpenVPN/openvpn/blob/fb4e8abca908d90faa98f20125e49a2590bccaa3/sample/sample-config-files/server.conf#L313-L315